tests/common/ixia/qos_fixtures.py

```diff
@@ -0,0 +1,104 @@
+import pytest
+"""
+RDMA test cases may require variety of traffic related fixtures. This file
+is repository of all different kinds of traffic related fixtures. This
+file currently holds the following fixture(s):
+    1. prio_dscp_map
+    2. all_prio_list
+    3. lossless_prio_list
+    4. lossy_prio_list
+"""
+
+@pytest.fixture(scope="function")
```

Comment:
we only need to do it once for the module. make the scope module for all the methods in this file